app/javascript/components/patient/close_contact/CloseContact.js

```diff
@@ -333,6 +366,15 @@ class CloseContact extends React.Component {
           </div>
         )}
         {this.state.showModal && this.createModal('Close Contact', this.toggleModal, this.submit)}
```

Comment:
I've been doing this as I see them, but you can remove these vars that are passed in to create modal and just put them in the createModal function

Reply:
Another update to this, when you change the title can you have the title say Add/Edit at the front depending on what it is? you can see how this is done for the submit button